aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -329,6 +334,52 @@ module aptos_framework::account {
         rotation_capability
     }
 
+    // Offer the account owner's signer capablity to another account
+    public entry fun offer_signer_capability_ed25519(account: &signer, signer_capability_sig_bytes: vector<u8>, account_public_key_bytes: vector<u8>, recipient_address: address) acquires Account {
```

Comment:
why do we want to use sig bytes?

Reply:
signature bytes is the signature by the account's private key on the SignerCapabilityOfferChallenge to indicates that the account owner intends to delegate their signer capability, so it's different from the signer